kong/include/kong/services/metrics/v1/README.md

```diff
@@ -0,0 +1,3 @@
+## Metrics
+
+Metrics service is used by a DP to report back its metrics to the CP.
```

Comment:
Please remove this service as well, this was not supposed to be part of OSS.

Reply:
done